Added project to Heroku (#2)
nishchenkokateryna/taxi-service: README.md

in the file from `/util/ConnectionUtil.java` to the ones you specified when installing MySQL
6. Also, you need change path in the `src/main/resources/log4j2.xml` at line `File name = "File" fileName = "logs\app.log"`
to absolute path to `.log` file
7. Finally, you can start the application

> ##Or you can just check it out following this link

Reviewer: ```suggestion
## Also, you can check it out following this link
```
